deploy/database/data.button.sql

```diff
@@ -1152,3 +1152,315 @@ UPDATE button
+UPDATE button SET flavor_text="Before becoming the Akami Professor of Internet Mathematics at UC San Diego, Dr. Fan Chung Graham directed math research labs at Bellcore. She and her husband Ron developed semi-random routing theories that keep the Internet running." WHERE name="Fan Chung Graham";
+UPDATE button SET flavor_text="Leonard Eugene Dickson loved pure mathematics. He wrote 267 research papers and 18 books, including the three volume \"History of the Theory of Numbers.\" His favorite quote: \"Thank God that number theory is unsullied by any applications.\"" WHERE name="Leonard Eugene Dickson";
+UPDATE button SET flavor_text="One of the best known and most respected mathematicians of the 20th century, Paul Erdos (air-dish) wrote or co-authored 1,475 papers. As he put it, \"A mathematician is a machine for turning coffee into theorems.\"" WHERE name="Paul Erdos";
+UPDATE button SET flavor_text="Pythagoras and his team in Italy called the mathematikoi developed cool triangle theorems, described the five regular solids, and thought that numbers had personal characteristics like male, female, beautiful, and ugly." WHERE name="Pythagoras";
+UPDATE button SET flavor_text="Known informally as \"The Mathemagician,\" Dr. Ron Graham was Chief Scientist at AT&T Labs. He's treasurer of the National Academy of Sciences, and also the ex-president of the International Jugglers Association." WHERE name="Ron Graham";
+UPDATE button SET flavor_text="Theano worked alongside her husband, Pythagoras, writing about mathematics, physics, medicine, and child psychology. She's best known for her work on the \"Golden Mean\" and the \"Golden Rectangle.\"" WHERE name="Theano";
+UPDATE button SET flavor_text="Tirade is a sentient broccoli. He is a master of time space and dimension, and has $50 dollars to spend on candy and gum. He also likes to beat people up." WHERE name="Tirade";
+UPDATE button SET flavor_text="The king's loyal nephew, he fights with the strength of three men." WHERE name="Gawaine";
+UPDATE button SET flavor_text="The Queen of the Court of Love, torn between her husband and her lover." WHERE name="Guenever";
+UPDATE button SET flavor_text="King of the Britons, the embodiment of Glory and Honor, and betrayed husband." WHERE name="King Arthur";
+UPDATE button SET flavor_text="The best knight in the world, torn between his love for the queen and duty to his king." WHERE name="Lancelot";
+UPDATE button SET flavor_text="The great magician who advises the king, but who always has his own agenda." WHERE name="Merlin";
+UPDATE button SET flavor_text="The king's bastard son, he has had villainy thrust upon him and revels in it." WHERE name="Mordred";
+UPDATE button SET flavor_text="The dark enchantress who beguiles all with mysterious faerie magic." WHERE name="Morgan Le Fay";
+UPDATE button SET flavor_text="The Lady of the Lake, she holds the powers of Excalibur and Avalon." WHERE name="Nimue";
```

Comment:
As in the case of Button Brains, these buttons ~~are kind of terrible~~ don't actually say "likes to beat people up" in their flavor text, and IMO should.